documentation/solution-architecture.md

```diff
@@ -0,0 +1,151 @@
+
+## Organisational layer
+
+The following stakeholders are identified:
+
+- issuer : automated (abundance) service available at location of issuer and/or through its website
+- clerk : employee that helps a client at location of issuer, services, identifies and authorizes the client manually
+- authorization service : automated service that identifies and authorizes client when client self services him/herself and accesses issuer through website
+- client : person in need of verifiable information required by a validator. It also becomes a holder of the resulting signed information.
+- validator : actor that needs to receive / verify information from client
+- administrator : actor that controls and monitors the automated issuer service
+
+An example of the essential conversation between real people as actors that would take place
+in the use cases the waardepapieren solution facilitates is given below:
+
+![](images/vcseq0.png)
+
+In essence this is the very proces that also takes place when retrieving a proof through the waardepapieren solution as can be seen below in the other verifiable claim sequence diagrams.
+
+![](images/vcseq.png)
+
+## Information layer
+
+All information is temporarily held in verifiable claim channels on the so called discipl ephemeral platform (http://github.com/discipl/discipl-core-ephemeral) in memory. This platform will enforce access rights denoted by the channel owner in specific allow claims within the channel, so by default all channels are only accessible to the channel owner holding a corresponding private key created and temporarily held by the channel owner.Of course the party hosting the ephemeral server and it's administrator needs to be trusted also, but as this is the issuer as source of the information it is by definition a party already trusted with this information by the client.
+
+The following channels will be created using this solution holding the following information (verifiable claims) with what kind of accessibility retention:
+
+1) public service channel of issuer, destroyed at service shutdown
+  - that a service with some id is attending to needs for what : (ssid, attendTo, what)
+  - (possibly a log of needs that has been attended to)
+2) public customer channel of client, destroyed after solving the need or at max for X minutes or service shutdown whatever comes first
+  - that an anonymous client with some id is in need for what : (ssid, need, what)
+3) private customer channel, destroyed after solving the need or at max for X minutes or service shutdown whatever comes first
+  - that the client refers to a certain BSN number for this. (ssid, BSN, bsn-number)
+  - that the client allows access to this channel to issuer
+  - that the client attests the information prepared by the issuer in the delivery channel to be true
```

Comment:
What does this mean? I don't see it in the diagram either

Reply:
changed it to conform to diagram... the attest of the prepared product has been removed and denoting the need as solved should be done in public client channel